### PR TITLE
Adding This Week in Rust Fest #5

### PR DESCRIPTION
**Pull request by gnunicorn**
Featuring:
- Rust Community Survey: Berlin is da capital!
- Graphs
- Update on the Talk Submissions
- Shut up and take my money gif
- Ancient Chinese God

### CODE REVIEW

File: _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+![Demographics](/assets/posts/twirf-5-locations.png)
+
+We knew that there was a need, but seeing these clear results... What can we say? Other than: **Your voice has been heard**. Loud and clear. We declare: There shall be a conference in Berlin, Capital of the Rust Community. It shall be on the 17th of September and it shall be called RUST FEST!
+
+
+## Reviewing Speaker Submissions
+
+As reported already last week, we were stunned by the submissions for talks, as well as for workshops and coaching (keep those coming!). The review panel has been busy going through all of them and we've established a short list already. A very exciting list, covering everything from hardware and hard algorithms, over real-world application of rust all the way to important discussions about rust as community and project of human beings. Unfortunately we still have to cut a few talks – there are just too many interesting topics and awesome proposals!!!
```
Comment (skade) [inline _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md]: rust -> Rust

File: _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "This Week in RustFest 5: Capital of the Rust Community, Speaker submissions and updates on tickets sales"
+socialImageSrc: '/assets/posts/twirf-5-locations.png'
+---
+
+## Berlin, Capital of the Rust Community
+
+A short while back the Rust Community Team ran a survey asking the participants about all sorts of things. Among others they asked about the demographics, where someone is living and that sort of thing. Just one day [after last weeks TWIRF](http://localhost:4000/blog/this-week-in-rustfest-4-closing-cfp-call-for-mentors) they [published the results on the rust blog](http://blog.rust-lang.org/2016/06/30/State-of-Rust-Survey-2016.html). With some remarkable results: a third of participants (1000 of 3000) said they are living in Europe and a whopping 40% **want a conference to take place in Berlin**. Whooot?
```
Comment (badboy) [inline _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md]: localhost url.

File: _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+We also contacted a few speakers to clarify some information. Once they got back to us, we'll compile a final schedule. We hope to have all speakers informed by mid next Week (the 14th) and will take logistics from there. We'll announce the first speakers soon after.
+
+
+## Can I has tickets yet?
+
+![Take my money](/assets/posts/twirf-5-take-my-money.gif)
+
+We understand you are psyched about all this. So are we. And in our original plan we wanted to have ticket sales going at this point in time already. As this is the first RustFest, however, there is a lot more administrative overhead this time around (legal entity, bank account and alike) and unfortunately KUI-XING, the [Chinese God of Bureaucracy]((http://www.godchecker.com/pantheon/chinese-mythology.php?deity=KUI-XING)), decided to play games with us: Certain things have been delayed more than necessary, other things become more complicated.
```
Comment (skade) [inline _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md]: Link broken (( )) -> ( )

File: _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md (hunk 1 above)
Comment (badboy) [inline _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md]: Do we really need three exclamation marks here?

File: _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md (hunk 3 above)
Comment (badboy) [inline _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md]: Double parentheses? Probably enough to wrap it once.

File: _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md (hunk 1 above)
Comment (gnunicorn) [inline _posts/2016-07-08-this-week-in-rustfest-5-we-are-rust-capital-please-stay-tuned.md]: YES !!!

**Pull request merged**